docs/support.md

```diff
@@ -1,3 +1,43 @@
-# Contact support
+---
+title: Support
+hide:
+  - navigation
+---
 
-TODO
+# How to get help
+
+We use a mixture of different platforms to provide support to Pack makers, so review the options below to determine the best way to get help.
+
+
+
+## Questions or advice
+
+For general questions about Packs, or advice on how to build them, we recommend you post in the [Coda Community "Packs" category][community_packs]. Codans actively monitor the community and answer questions, as well as other Pack makers that have experience with the platform. Posting questions and answers in a public forum helps build up a knowledge base of information that helps everyone.
+
+<form action="https://community.coda.io/search">
+  <p>
+    <input type="text" name="q" id="q" value="#developers-central:coda-packs " class="md-input md-input--stretch"/>
+  </p>
+  <p>
+    <button type="submit" class="md-button">Search existing questions</button>
+    <a href="https://community.coda.io/new-topic?category=developers-central/coda-packs" class="md-button md-button--primary">Post a new question</a>
+  </p>
+</form>
+
+
+## Problems or bugs
+
+If you are experiencing a problem with your Pack or have identified a bug you should reach out to the Coda Support team. Send an email to [help+packs@coda.io][support_email], and include the following information:
+
+- The ID of your Pack (visible in the Pack Studio URL).
```

Comment:
Could also just say to send the url of your pack (packs studio url and gallery listing url would both have the id in it). Seems people generally have a hard time figuring out how mentally to extract content from urls even if it seems obvious.

Reply:
Good call, updated to mention the Pack Studio URL to make it explicit.